src/components/Reports/ReportsNew.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { Helmet } from 'react-helmet';
+
+const ReportsNew = () => {
+  return (
+    <>
+      <FormattedMessage id="reports_title">
+        {(reports_title) => (
+          <Helmet>
+            <title>{reports_title}</title>
+          </Helmet>
+        )}
+      </FormattedMessage>
+      <section className="dp-container" style={{ marginTop: '20px' }}>
```

Comment:
is it fine hardcoding the styles here?

CC: @GusBaamonde

Reply:
I mention it in my previous comment

Reply:
fixed